.github/workflows/build-test.yml

```diff
@@ -6,8 +6,8 @@ on:
 
 
 jobs:  
-  build:
-    name: Test Builds
+  build-linux:
```

Comment:
If no custom steps are needed I guess we can use a matrix strategy like the following one:
```
runs-on: ${{ matrix.os }}
strategy:
    matrix:
        os: [ubuntu-latest, windows-latest, macOS-latest]
```

Reply:
Please also add `macos-11`, because `macos-latest` only covers `Catalina`.

Reply:
Done